app/src/processing/app/EditorTab.java

```diff
@@ -148,6 +148,7 @@ private SketchTextArea createTextArea(RSyntaxDocument document)
     textArea.setMarginLineEnabled(false);
     textArea.setCodeFoldingEnabled(PreferencesData.getBoolean("editor.code_folding"));
     textArea.setAutoIndentEnabled(PreferencesData.getBoolean("editor.indent"));
+    textArea.setCloseCurlyBraces(PreferencesData.getBoolean("editor.indent"));
```

Comment:
Agree, it should be moved to its own preference. I added it this way to check if the functionality was ok and provide a build with it